modules/blocks.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Load code specific to Gutenberg blocks which are not tied to a module.
+ * This file is unusual, and is not an actual `module` as such.
+ * It is included in ./module-extras.php
```

Comment:
Since the file is loaded via this list, it will be loaded on all Jetpack sites, whether they are connected to WordPress.com or not. Maybe we should take advantage of this function to make sure we don't actually attempt to register blocks when they should not be registered:
https://github.com/Automattic/jetpack/blob/544516b71858915698e1f48e8f6db63689f8ebbb/class.jetpack-gutenberg.php#L90

Reply:
Sure, so the proposed solution would be to remove `blocks.php` from `$tools`, and then add this below?

```
if ( Jetpack_Gutenberg::should_load_blocks() ) {
	$tools[] = 'blocks.php';
}
```

Reply:
Right, that's an approach we could take I think.

Reply:
Done: https://github.com/Automattic/jetpack/pull/10503/commits/076cb38f8f01a4070156bb99b8676c24d2afef40